### DISCUSSION TITLE
MFCMAPI configuration options with Office 365

### DISCUSSION THREAD

**Discussion by Navoxctr** [microsoft/mfcmapi#484] [Q&A]
Hey there! 

I was wondering if there are any specific options I need to checkmark in order to use MFCMAPI 32bit once again. 

Every time there is an office update it seems to break mfcmapi and I am a little frustrated because either in live mode or cached mode, with the outlook client open first or not, I cannot get mfcmapi to load anymore. 

I have to use an older outlook client like 2013 in order to get MFCMAPI to load my mailbox. 

Just curious if there are any options or anything I need to configure in order to use mfcmapi once again or if its just broken with office 365 versions. Thank you.

**Comment by stephenegriffin** [comment]
Turn off the bypass cache options, start Outlook and access the target profile once to cache auth, and use the same bitness MFCMAPI as Outlook. It should work then. If it doesn't, then you'll need to be more specific about what you're doing.
